.github/workflows/build.yml

```diff
@@ -17,16 +17,18 @@ jobs:
       run: |
         sudo apt update -y
         sudo apt install -y libgtest-dev libgflags-dev openssl libssl-dev protobuf-compiler protoc-gen-go golang-go cmake
-    - name: make
+
+    - name: test-core-certifier-programs
```

Comment:
Refactored existing tests in build.yml into named chunks of 'run'-workflows. This makes testing and results more granular.